kerastuner/engine/tuner_utils.py

```diff
@@ -75,145 +76,35 @@ def get_max_epochs_and_steps(fit_args, fit_kwargs):
 
 class TunerCallback(keras.callbacks.Callback):
 
-    def __init__(self, tuner, trial, execution):
+    def __init__(self, tuner, trial):
         self.tuner = tuner
         self.trial = trial
-        self.execution = execution
 
     def on_epoch_begin(self, epoch, logs=None):
         self.tuner.on_epoch_begin(
-            self.execution, self.model, epoch, logs=logs)
+            self.trial, self.model, epoch, logs=logs)
```

Comment:
I don't understand this -- a callback is for a training run, which is an execution. A Trial may cover multiple executions. Trial == unique set of hp values.

Reply:
Discussed offline, added MultiExecutionTuner to handle execution logic